lhotse/dataset/vad.py

```diff
@@ -0,0 +1,55 @@
+import logging
+from pathlib import Path
+from typing import Dict, Optional
+
+import torch
+from torch.utils.data import Dataset
+
+from lhotse.cut import CutSet
+from lhotse.utils import Pathlike
+
+EPS = 1e-8
+
+
+class VadDataset(Dataset):
+    """
+    The PyTorch Dataset for the voice activity detection task.
+    Returns a dict of:
+
+    .. code-block::
+
+        {
+            'features': (T x F) tensor
+            'is_voice': bool
```

Comment:
I think this should be a 1-dim tensor of 0/1 values for each feature frame. Let's say you have a `Cut` of duration 10 seconds, with a supervision between second 3 and 7 - then you would return sth like `torch.tensor(300 * [0] + 400 * [1] + 300 * [0])` with frame_shift = 10ms

Reply:
Essentially, instead of a "sequence classification" task, we want a "sequence tagging" or "frame classification" task, which is how VAD would usually be used.